### ISSUE TITLE
[MU4 Issue] Change to part name gets reverted after file close/open

### ISSUE THREAD

**Issue by SamYeh** [musescore/MuseScore#14531]
**Describe the bug**
Modification to part name does not persist after the file is saved, closed, and reopened. It gets reverted to the default part name.

**To Reproduce**
1. Create a new score.
2. Change the name(s) of part(s).
3. Click the Parts button to verify the updated part name(s).
4. Save, close, and reopen the file.
5. Click the Parts button. Observe the part name(s) being reverted to the default.

**Expected behavior**
Changes to part name(s) should persist.

As a side note, the Staff/Part properties dialog box no longer contains the Part name field. Is that by design?
![image](https://user-images.githubusercontent.com/10248807/201830563-a9bf30b7-89e0-4e35-bed8-54b0fea80f20.png)

**Platform information**
 - OS: Windows 11

**Comment by DmitryArefiev** [comment]
Сan reproduce on my side as well. The issue **occurs if parts were not opened** (generated) **previously**.


https://user-images.githubusercontent.com/90187801/201872963-e37d7b03-1cd1-4563-a42b-a251c7cf1d9d.mp4


Thanks for logging!

**Comment by DmitryArefiev** [comment]
> As a side note, the Staff/Part properties dialog box no longer contains the Part name field. Is that by design?

Yes, that was removed by design because of complex dependency between changing that field and changing real part name

**Comment by sammik** [comment]
> The issue occurs if parts were not opened (generated) previously.

But if they were, Part name in part is not updated (should be 1, 2, 3, 4 and is Violin I, ...). 
Is this logged already?

**Comment by DmitryArefiev** [comment]
> > The issue occurs if parts were not opened (generated) previously.
> 
> But if they were, Part name in part is not updated (should be 1, 2, 3, 4 and is Violin I, ...). Is this logged already? EDIT: Yes, it is #14502

Yeah!
